src/jit/gentree.h

```diff
@@ -1333,7 +1333,11 @@ struct GenTree
     bool OperIsMultiRegOp() const
     {
 #if !defined(LEGACY_BACKEND) && defined(_TARGET_ARM_)
-        if (gtOper == GT_MUL_LONG || gtOper == GT_PUTARG_REG || gtOper == GT_COPY)
+        if (gtOper == GT_MUL_LONG ||
+#ifdef SOFT_FP
```

Comment:
This should be `#ifdef ARM_SOFTFP`.

And I think `GT_COPY` is MultiRegOp on armel only as well as `GT_PUTARG_REG` since these two nodes come together for double register argument passing.

Reply:
Thanks for catching the typo! I believe that `GT_COPY` should be enabled for both. At present, the register allocator doesn't insert copies of tree temps, but it may in future (e.g. to support conflicting register requirements, or to avoid a spill), in which case it will be needed if a `GT_MUL_LONG` is copied.